libs/designsystem/src/lib/components/checkbox/checkbox.component.ts

```diff
@@ -1,51 +1,37 @@
-import {
-  ChangeDetectionStrategy,
-  Component,
-  EventEmitter,
-  Input,
-  OnChanges,
-  Output,
-  SimpleChange,
-  SimpleChanges,
-} from '@angular/core';
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
 
 @Component({
   selector: 'kirby-checkbox',
   templateUrl: './checkbox.component.html',
   styleUrls: ['./checkbox.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class CheckboxComponent implements OnChanges {
+export class CheckboxComponent {
   @Input() checked: boolean;
   @Input() disabled: boolean;
-  @Input() color: string = 'primary';
-  @Input() shape: string = 'square';
+  @Input() error: boolean;
```

Comment:
Consider aligning naming with other components (e.g. `Dropdown`, `Input`, `Textarea`):
```suggestion
  @Input() hasError: boolean;
```